src/pages/en/guides/troubleshooting.md

```diff
@@ -70,7 +70,7 @@ Then check your import statement:
 
 - Does your import have the same name as the imported component? (Check your component name and that it [follows the `.astro` syntax](/en/comparing-astro-vs-other-tools/#astro-vs-jsx).)
 
-- Have you included the extension in the import? (Check that your imported file contains an extension. e.g. `.astro`, `.md`, `.jsx`, `.vue`)
+- Have you included the extension in the import? (Check that your imported file contains an extension. e.g. `.astro`, `.md`, `.vue`, `.svelte`)
```

Comment:
```suggestion
- Have you included the extension in the import? (Check that your imported file contains an extension. e.g. `.astro`, `.md`, `.vue`, `.svelte`. Note: no file extension is required for `.jsx` files.)
```

Reply:
Do we need to also mention `.tsx`?

```suggestion
- Have you included the extension in the import? (Check that your imported file contains an extension. e.g. `.astro`, `.md`, `.vue`, `.svelte`. Note: no file extension is required for `.jsx` and `.tsx` files.)
```

Reply:
Sounds good. Given that this section is specifically about importing **components**, I'm assuming we don't need to worry about mentioning `.js`, `.ts` here.